tests/model_hub_tests/models_hub_common/constants.py

```diff
@@ -16,10 +17,12 @@
 '''
 runtime_heat_duration = os.environ.get('RUNTIME_HEAT_DURATION', '5')
 
-
 tf_hub_cache_dir = os.environ.get('TFHUB_CACHE_DIR',
                                   os.path.join(tempfile.gettempdir(), "tfhub_modules"))
+hf_cache_dir = os.environ.get('HF_CACHE_DIR',
+                              os.path.join(tempfile.gettempdir(), "hugging_face"))
 os.environ['TFHUB_CACHE_DIR'] = tf_hub_cache_dir
+os.environ['HF_HOME'] = hf_cache_dir
```

Comment:
```suggestion
os.environ['HF_CACHE_DIR'] = hf_cache_dir
```
?

Reply:
Agree.

Reply:
Changed to `HF_HUB_CACHE`, according to this doc: https://huggingface.co/docs/huggingface_hub/package_reference/environment_variables#hfhubcache